src/Featurizers/Components/UnitTests/DocumentStatisticsEstimator_UnitTest.cpp

```diff
@@ -6,8 +6,9 @@
 #include "catch.hpp"
```

Comment:
I'll add comments to this file once the interface has stabilized. Please make sure that all scenarios are covered in tests.